### DISCUSSION TITLE
Review

### DISCUSSION THREAD

**Discussion by galalkoro98** [HYF-Class19/agile-development-group-1#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by sammou00** [comment]
Hello There, Good job.
Some Notes :
1- [constrains](https://github.com/HYF-Class19/agile-development-group-1/blob/master/planning/constraints.md)
You have mentioned the definition of the constrains External and eternal, which it is not bad , but what we want really is just what constrains you may facing during building you project.
ex : 
### External : 
- One week deadline.
 ### Internal 
- Team availability or different members have different time availably, which doesn't match for the whole team, which make it difficult to meet and working on the project planning  

2- [ user personas](https://github.com/HYF-Class19/agile-development-group-1/blob/master/planning/user-personas.md)
Well, you user personas has nothing to do with your project , even remotely.

- Student :  

-      `Bio` :  John Doe is  having a biology class and need to write an essay about trees .
-       `Need/Goals` : he need to gather some information about trees  to write he essay.
-      `Frustrations` : most other resource are too advance for him to understand , he is looking for some resources which can provides a 
         basic knowledge about trees  first.
-      `Scenario ` : John would like to write a good essay , but he has no basic information about tress and he doesn't have much time as he 
        has other classes  to follow, so he is trying to do his best with the time he has.

Now, this user personas is related to my project all-about-Trees, compare this user to your user personas .

We will discus more in the class
